features/step_definitions/animal_steps.rb

```diff
@@ -1 +1,11 @@
-#Delete this comment, here is where you should write your step defs
+Given(/^a animal that is a dog$/) do
+  @animal = Animal.new('Rex', 'dog', 5)
+end
+
+When(/^he is (\d+)$/) do |age|
+  expect(@animal.age.to_s).to eq age
+end
+
+Then(/^he is old$/) do
+  expect(@animal.old?).to be true
```

Comment:
https://www.relishapp.com/rspec/rspec-expectations/v/3-2/docs/built-in-matchers/predicate-matchers